src/components/KTreeList/KTreeItem.vue

```diff
@@ -67,64 +67,31 @@
   </div>
 </template>
 
-<script lang="ts">
-import type { PropType } from 'vue'
-import { computed, useSlots } from 'vue'
-import type { TreeListItem } from '@/types'
-
-export const itemsHaveRequiredProps = (items: TreeListItem[]): boolean => {
-  return items.every(i => i.name !== undefined && i.id !== undefined && (!i.children?.length || itemsHaveRequiredProps(i.children)))
-}
```

Comment:
This was removed because TypeScript already performs this check. Recursively verifying it in every node would be excessive.